sdcm/nemesis.py

```diff
@@ -3223,7 +3223,8 @@ def _shrink_cluster(self, rack=0):
         InfoEvent(message=f'Start shrink cluster by {add_nodes_number} nodes').publish()
         # Check that number of nodes is enough for decommission:
         cur_num_nodes_in_dc = len([n for n in self.cluster.nodes if n.dc_idx == self.target_node.dc_idx])
-        initial_db_size = self.tester.params.get("n_db_nodes")
+        initial_config_name = "k8s_n_scylla_pods_per_cluster" if self._is_it_on_kubernetes() else "n_db_nodes"
+        initial_db_size = self.tester.params.get(initial_config_name)
```

Comment:
In case of K8S the `k8s_n_scylla_pods_per_cluster` config option is not required. So, it may be empty.
Even if we set the defaults it is better to have safer coding.
So, I propose to do following:

```
initial_db_size = self.tester.params.get("n_db_nodes")  # this line stays unchanged
if self._is_it_on_kubernetes():
    initial_db_size = self.tester.params.get("k8s_n_scylla_pods_per_cluster") or initial_db_size
```

Reply:
I think not having `k8s_n_scylla_pods_per_cluster` defined would fail on cluster creation

but I'm o.k. with something like this:
```python
initial_db_size = self.tester.params.get("n_db_nodes")  # this line stays unchanged
if self._is_it_on_kubernetes():
    initial_db_size = self.tester.params.get("k8s_n_scylla_pods_per_cluster",  initial_db_size)
```